lib/run-status.js

```diff
@@ -166,6 +172,7 @@ RunStatus.prototype.processResults = function (results) {
 	this.tests = flatten(this.tests);
 
 	this.passCount = sum(this.stats, 'passCount');
+	this.knownFailureCount = sum(this.stats, 'knownFailureCount');
```

Comment:
Seems that the results does not have that count, I grep `processResults` in directory but find nothing, can you tell me where it is called?